ConsoleApp/School/Student.cs

```diff
@@ -0,0 +1,7 @@
+namespace ConsoleApp.School;
+
+class Student : Person
+{
+        public string gender { get; set;}
```

Comment:
code convention violated here
```suggestion
        public string Gender { get; set;}
```